ManPac/Assets/Scripts/Player/Death/DeathHandler.cs

```diff
@@ -40,6 +40,19 @@ public void CallDeath()
         StartCoroutine(Death());
     }
 
+    public void ChangeGhostMovementSpeed(float slowSpeed)
+    {
+        if (_isInDanger)
+        {
+            _thisPlayerTraverser.CurrentVelocity = slowSpeed;
+        }
+
+        if (!_isInDanger)
+        {
+            _thisPlayerTraverser.CurrentVelocity = 1f;
+        }
```

Comment:
you can make this a turnery operator
```suggestion
        if (_isInDanger)
        {
            _thisPlayerTraverser.CurrentVelocity = slowSpeed;
        }

        if (!_isInDanger)
        {
            _thisPlayerTraverser.CurrentVelocity = 1f;
        }
        to 
         _thisPlayerTraverser.CurrentVelocity = _isInDanger?
                slowSpeed : 1f;
```